internal/lsp/messages.go

```diff
@@ -120,6 +120,15 @@ type TextDocumentIdentifier struct {
 	URI string `json:"uri"`
 }
 
+type TextDocumentDidChangeParams struct {
```

Comment:
This type represents a messages that is sent in the protocol and so should have been here to begin with. it has been moved.